hermes-schema/src/main/java/pl/allegro/tech/hermes/schema/CachedSchemaVersionsRepository.java

```diff
@@ -42,17 +40,19 @@ public CachedSchemaVersionsRepository(RawSchemaClient rawSchemaClient, ExecutorS
     }
 
     @Override
-    public List<SchemaVersion> versions(Topic topic, boolean online) {
+    public SchemaVersionsResponse versions(Topic topic, boolean online) {
         try {
             if (online) {
                 List<SchemaVersion> v = rawSchemaClient.getVersions(topic.getName());
                 versionsCache.put(topic, v);
-                return v;
+                return SchemaVersionsResponse.succeeded(v);
```

Comment:
v -> versions? Since in the else branch it is fully named.

Reply:
Done